suite-native/module-send-receive/src/components/ReceiveTextHint.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Button, VStack, Text } from '@suite-native/atoms';
+
+type ReceiveTextHintProps = {
+    onShowAddress(): void;
+};
+
+const hintWrapperStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.medium,
+    alignItems: 'center',
+    textAlign: 'center',
+}));
+
+const hintTextStyle = prepareNativeStyle(_ => ({
+    textAlign: 'center',
+}));
+
+export const ReceiveTextHint = ({ onShowAddress }: ReceiveTextHintProps) => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <VStack spacing="large">
+            <VStack spacing="medium" style={applyStyle(hintWrapperStyle)}>
+                {/* TODO: change the texts when the final design is ready. */}
+                <Text variant="titleSmall">
+                    <Text variant="titleSmall" color="forest">
+                        Trezor Go
+                    </Text>{' '}
+                    addresses
+                </Text>
```

Comment:
Doesn't inner text inherit variant (font size, width etc)?

Reply:
I think there is default variant=body for all text, so probably no

Reply:
Yes, it does not.

Reply:
Would be nice to have this inherit behavior, but not big deal. We can create task for backlog.

Reply:
Issue created: https://github.com/trezor/trezor-suite/issues/7468